### PR TITLE
Fix PyPI packaging

### LINKED ISSUE #253: mlagility PyPI package doesn't have any source code in it
The mlagility 2.1.2 package on PyPI looks like this internally...

image

...meaning it is missing all of the source code.

We must be messing something up during packaging. Setting p0 since this could really confuse a new user. 

We may want to figure out a way to keep PyPI packages under test as well.

### PR DESCRIPTION
**Pull request by jeremyfowers**
Closes #253 

Fixes a variety of bugs/problems with our PyPI packaging, including:
- mlagility and onnxflow source code not included in the package
- benchmarking Dockerfile not included in the package
- models corpus not included in the package
- no good way to programmatically determine the models directory location on the CLI
- no testing for the PyPI wheel
- we forgot to bump the major version number in #241 
- not obvious how to run the `benchit` command in the BERT tutorial

### CODE REVIEW

File: .github/workflows/test_mlagility.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ jobs:
         shell: bash -el {0}
         run: |
           pylint src/mlagility --rcfile .pylintrc
-          pylint examples --rcfile .pylintrc --ignore-paths examples/onnxflow
+          pylint examples --rcfile .pylintrc --ignore-paths examples/onnxflow --disable E0401,E0611
```
Comment (danielholanda) [inline .github/workflows/test_mlagility.yml:43]: Isn't it a better solution to remove `import-error` and `no-name-in-module` from `.pylintrc`?

File: src/mlagility/common/filesystem.py [hunk 2]
```diff
@@ -23,10 +24,7 @@
 BUILD_MARKER = ".mlabuild"
 
 # Locate the models directory
-# This file is located in MLAGILITY_INSTALL/src/mlagility/common/
-# The models directory is located at MLAGILITY_INSTALL/models
-# This relative positioning is protected by the "test_000_models_dir" in test/unit.py
-MODELS_DIR = pathlib.Path(__file__).parents[3] / "models"
+MODELS_DIR = importlib.util.find_spec("mlagility_models").submodule_search_locations[0]
```
Comment (danielholanda) [inline src/mlagility/common/filesystem.py:27]: Got the following error when trying this by cloning and pip installing:

```
Traceback (most recent call last):
  File "/net/home/dhnoronha/miniconda3/envs/atlas/bin/benchit", line 33, in <module>
    sys.exit(load_entry_point('mlagility', 'console_scripts', 'benchit')())
  File "/net/home/dhnoronha/miniconda3/envs/atlas/bin/benchit", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/net/home/dhnoronha/miniconda3/envs/atlas/lib/python3.8/importlib/metadata.py", line 77, in load
    module = import_module(match.group('module'))
  File "/net/home/dhnoronha/miniconda3/envs/atlas/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/net/home/dhnoronha/mlagility/src/mlagility/__init__.py", line 3, in <module>
    from .api.script_api import benchmark_script
  File "/net/home/dhnoronha/mlagility/src/mlagility/api/script_api.py", line 10, in <module>
    import mlagility.cli.slurm as slurm
  File "/net/home/dhnoronha/mlagility/src/mlagility/cli/slurm.py", line 7, in <module>
    import mlagility.common.filesystem as filesystem
  File "/net/home/dhnoronha/mlagility/src/mlagility/common/filesystem.py", line 27, in <module>
    MODELS_DIR = importlib.util.find_spec("mlagility_models").submodule_search_locations[0]
AttributeError: 'NoneType' object has no attribute 'submodule_search_locations'
```
When changing it to `MODELS_DIR = pathlib.Path(__file__).parents[3] / "models"` things work as expected.

Comment (danielholanda) [reply src/mlagility/common/filesystem.py:27]: pip installing again solves it!

**Review by danielholanda — APPROVED** [review]

File: .github/workflows/test_mlagility.yml (hunk 1 above)
Comment (jeremyfowers) [reply .github/workflows/test_mlagility.yml:43]: pylintrc is global for the whole project. I only disabled those errors for this specific folder.

**Comment by jeremyfowers** [comment]
FYI @jcwchen @ramkrishna2910 @danielholanda 

If you installed `mlagility` with `pip install -e`, you will need to re-install after pulling this commit.

**Pull request merged**